### DISCUSSION TITLE
Как оставить кнопку в topbuttons?

### DISCUSSION THREAD

**Discussion by IrinaG1090** [totumonline/totum-ru-issues-and-discussions#450] [Есть идея, а что если...]
Добрый день, подскажите пожалуйста, вот такой момент.

в настройках таблицы есть такая строка:
=: setFormat (topbuttons: false)

Как используя эту настройку оставить видимую кнопку 
![Снимок экрана от 2023-10-05 15-12-58](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/23610956/8969c7d9-d81f-4708-8fe8-3d5bf2488a25)
которая переключает таблица <=> канбан, панели

**Comment by ttm-alexeypolunin** [comment]
Никак, они в одном блоке. Я запишу, может когда-нибудь ее оттуда достанут.

**Reply by IrinaG1090** [reply]
Поняла. 
Тогда как прописать отображение цикла? Можно ли цикл показывать автору цикла только в виде таблицы, а у других у кого есть доступ к циклу могли видеть и в виде таблицы и в виде канбан? или может есть другое какое решение?

**Reply by ttm-alexeypolunin** [reply]
Эта настройка не является динамической и не чуствительна к любым параметрам доступа.

**Reply by IrinaG1090** [reply]
поняла, спасиб
